lib/mail_room/crash_handler.rb

```diff
@@ -2,18 +2,15 @@
 module MailRoom
   class CrashHandler
 
-    attr_reader :error, :format
+    SUPPORTED_FORMATS = %w[json plain]
 
-    SUPPORTED_FORMATS = %w[json none]
-
-    def initialize(error:, format:)
-      @error = error
-      @format = format
+    def initialize(stdout=STDOUT)
```

Comment:
If someone passes in STDERR, then we have `@stdout = STDERR` :-P Do you mind renaming the variable `@stdout` to something else?

Reply:
oh, great catch!

Reply:
Changed to `stream`. WDYT?

Reply:
💯